OpenRA.Mods.Common/UtilityCommands/UpgradeRules.cs

```diff
@@ -270,6 +270,16 @@ internal static void UpgradeActorRules(ModData modData, int engineVersion, ref L
 					}
 				}
 
+				if (engineVersion < 20160717)
```

Comment:
It is still a few days above the `20160713` entry. Let's please not time travel even when reviews take forever.